gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -50,41 +51,36 @@ public class GcpLogSink implements Sink {
 
   private static final String DEFAULT_LOG_NAME =
       "microservices.googleapis.com%2Fobservability%2Fgrpc";
+  private static final Severity DEFAULT_LOG_LEVEL = Severity.DEBUG;
   private static final String K8S_MONITORED_RESOURCE_TYPE = "k8s_container";
   private static final Set<String> kubernetesResourceLabelSet
       = ImmutableSet.of("project_id", "location", "cluster_name", "namespace_name",
       "pod_name", "container_name");
-  private static final long FALLBACK_FLUSH_LIMIT = 100L;
   private final String projectId;
   private final Map<String, String> customTags;
   private final MonitoredResource kubernetesResource;
-  private final Long flushLimit;
   /** Lazily initialize cloud logging client to avoid circular initialization. Because cloud
    * logging APIs also uses gRPC. */
   private volatile Logging gcpLoggingClient;
-  private long flushCounter;
   private final Collection<String> servicesToExclude;
 
   @VisibleForTesting
   GcpLogSink(Logging loggingClient, String destinationProjectId, Map<String, String> locationTags,
-      Map<String, String> customTags, Long flushLimit, Collection<String> servicesToExclude) {
-    this(destinationProjectId, locationTags, customTags, flushLimit, servicesToExclude);
+      Map<String, String> customTags, Collection<String> servicesToExclude) {
+    this(destinationProjectId, locationTags, customTags, servicesToExclude);
```

Comment:
I followed how locationTags are used and it looks like only `project_id` from those tags is used and everything else is discarded. Is that true? In that case most of the code in `GlobalLocationTags` is redundant?

Reply:
No, we use `GlobalLocationTags` to populate [Monitored Resource](https://github.com/grpc/grpc-java/blob/18602b74e144efb731158ae687f09b42d0a814bc/gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java#L162) for every log entry.

Reply:
Looking at `io.grpc.gcp.observability.logging.GcpLogSink.getCustomTags()` I don't see locationTags being used except to find source_project_id. And I don't see any other refs to `GlobalLocationTags.getLocationTags()`

Reply:
References to `GlobalLocationTags.getLocationTags()` are not seen because the result is passed directly to [GcpLogSink](https://github.com/grpc/grpc-java/blob/master/gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java#L81) as `locationTags` and [used](https://github.com/grpc/grpc-java/blob/master/gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java#L85) in creating MonitoredResource object.

Reply:
You are right - I missed one reference.